WalletWasabi.Backend/README.md

```diff
@@ -36,7 +36,7 @@
   
 #### Handling Reorgs
 
-  If the answer to the `filters` request is not found, then the client steps back one block and queries the filters with that previous hash. This can happen multiple times. This will only happen when blockchain reorganization happened. 
+  If the answer to the `filters` request is not found, then the client steps back one block and queries the filters with that previous hash. This can happen multiple times. This will only happen when blockchain reorganization happen. 
```

Comment:
> This will only happen when blockchain reorganization happen 

Should be "happens" (at the end). Even better would be imo: "This will only happen if a blockchain reorganization has occured"

Reply:
Good catch. Fixed it.